controllers/auth.js

```diff
@@ -29,9 +29,34 @@ ROUTER.post('/login', (req, res) => {
     });
 });
 // POST /auth/signup
-// We need to decide how we want this post route to work
-// I'm currently listing ways we can reduce human error
-
+ROUTER.get('/signup', (req, res) => {
```

Comment:
ROUTER.post?

Reply:
should be fixed and pull request submiited